autotest.pm

```diff
@@ -147,13 +142,13 @@ sub runalltests {
                 }
             }
             else {
-                if ($snapshots_supported && $flags->{milestone}) {
+                if ($snapshots_supported && ($flags->{milestone} || $bmwqemu::vars{MAKETESTSNAPSHOTS})) {
                     make_snapshot('lastgood');
```

Comment:
eg. sometimes I'd like to jump to a step before system settings changed then doing debugging